subprojects/test-kit/src/main/java/org/gradle/testkit/runner/internal/ToolingApiGradleExecutor.java

```diff
@@ -110,10 +112,12 @@ public GradleExecutionResult run(GradleExecutionParameters parameters) {
             connection = gradleConnector.connect();
             targetGradleVersion = determineTargetGradleVersion(connection);
             if (targetGradleVersion.compareTo(TestKitFeature.RUN_BUILDS.getSince()) < 0) {
-                throw new UnsupportedFeatureException(String.format("The version of Gradle you are using (%s) is not supported by TestKit. TestKit supports all Gradle versions %s and later.",
-                    targetGradleVersion.getVersion(), TestKitFeature.RUN_BUILDS.getSince()));
+                throw new UnsupportedFeatureException(String.format("The version of Gradle you are using (%s) is not supported by TestKit. TestKit supports all Gradle versions 2.6 and later.", targetGradleVersion.getVersion()));
```

Comment:
Just to be explicit, I'd nudge old users to try to update straight to 3.0.
```suggestion
                throw new UnsupportedFeatureException(String.format("The version of Gradle you are using (%s) is not supported by TestKit. TestKit supports all Gradle versions 3.0 and later.", targetGradleVersion.getVersion()));
```